### PR TITLE
Add start script to package.json

### PR DESCRIPTION
**Pull request by DimaProkop**
Add command **npm start**  in package.json (field _script:start_)

### CODE REVIEW

**Comment by KrivichaninDenis** [comment]
update description

File: package.json [hunk 1]
```diff
@@ -16,12 +16,11 @@
     "url": "https://github.com/pub-t/plain-destination/issues"
   },
   "homepage": "https://github.com/pub-t/plain-destination#readme",
-  "devDependencies": {
-    "eslint": "^2.12.0",
```
Comment (irnc) [inline package.json]: Why you deleted it?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "bus-client",
+  "name": "plain-destination",
```
Comment (irnc) [inline package.json]: Name and description should be changed using separate pull request.

File: package.json [hunk 3]
```diff
@@ -16,12 +16,11 @@
...
-    "eslint-plugin-jsx-a11y": "^1.4.2",
-    "eslint-plugin-react": "^5.1.1"
-  },
-  "dependencies": {}
+  "devDependencies": {},
+  "dependencies" : {
+     "http-server" : "latest"
+   },
+  "scripts": {
+    "start": "http-server [path]"
```
Comment (irnc) [inline package.json]: Have you tried to run this as `npm start`? I guess every request to that server would fail because `[path]` is not a directory.

File: package.json [hunk 4]
```diff
@@ -16,12 +16,11 @@
...
-    "eslint": "^2.12.0",
-    "eslint-config-airbnb": "^9.0.1",
-    "eslint-plugin-import": "^1.8.1",
-    "eslint-plugin-jsx-a11y": "^1.4.2",
-    "eslint-plugin-react": "^5.1.1"
-  },
-  "dependencies": {}
+  "devDependencies": {},
+  "dependencies" : {
+     "http-server" : "latest"
```
Comment (irnc) [inline package.json]: `latest` is not a valid semantic version, you should use `npm i --save http-server` to add dependency properly.

**Comment by irnc** [comment]
This pull request needs additional work and should be split in two, for issues #3 and #4.
